infra/docs.sh

```diff
@@ -16,5 +16,5 @@ cp dev/docs/google2ed1af765c529f57.html dev/docs/doc
 
 # Upload the docs.
 if [ "$1" = "--upload" ]; then
-  gsutil -m rsync -d -r dev/docs/doc/ gs://docs.flutter.io/
+  gsutil -m cp -z "js,json,html,css" dev/docs/doc/ gs://docs.flutter.io/
```

Comment:
The problem with orphans is that they're still live links.  Instead of getting a 404s, people will get outdated (i.e., wrong and harmful) documentation.